invoice_comment_template/views/base_comment_template_view.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
   <data noupdate="0">
```

Comment:
Please remove data tag since it has noupdate="0", see [Migration-to-10 guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0)